policy/system/auth.cas

```diff
@@ -436,10 +436,10 @@ module auth_extra {
 }
 
 module pam {
-    domain chkpwd_t;
+    resource chkpwd_t;
```

Comment:
The declaration needs to be fixed instead. This is definitely not a resource.

Reply:
fixed, thanks.